### ISSUE TITLE
jCommander 1.75 causes build issues

### ISSUE THREAD

**Issue by timmattison** [awslabs/aws-iot-client#6]
When building with jCommander 1.75 this error shows up:

```
> Task :compileJava
error: error reading /Users/timmatt/.gradle/caches/modules-2/files-2.1/com.beust/jcommander/1.75/e89c678b272c31955bff08b5c912d70dc6e5dc4c/jcommander-1.75.jar; invalid manifest format
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
```

This was fixed in 1.74 and regressed in 1.75.

**Comment by timmattison** [comment]
https://github.com/cbeust/jcommander/issues/434 - link to jCommander issue

**Comment by cbeust** [comment]
Should be fixed in JCommander 1.78.
